R/build-favicons.R

```diff
@@ -65,42 +64,37 @@ build_favicons <- function(pkg = ".", overwrite = FALSE) {
       )
     )
   )
-
-  resp <- httr::RETRY(
-    "POST",
-    "https://realfavicongenerator.net/api/favicon",
-    body = json_request,
-    encode = "json",
-    quiet = TRUE
+  req <- httr2::request("https://realfavicongenerator.net/api/favicon")
+  req <- httr2::req_body_json(req, json_request)
+
+  withCallingHandlers(
+    resp <- httr2::req_perform(req),
+    error = function(e) {
+      cli::cli_abort("API request failed.", parent = e)
```

Comment:
indicate which API?

Reply:
I contemplated that too, but it ends up just echoing the url in the previous message, and the error will have `build_favicon()` in it, so I think it's pretty clear. (Plus I've never actually seen this fail, which is pretty remarkable)